resources/tomcat-config/crafter/studio/extension/studio-config-override.yaml

```diff
@@ -116,10 +116,9 @@ studio.db.socket: /tmp/MariaDB4j.@MARIADB_PORT@.sock
 # studio.authentication.headers.groups: groups
 # Enable/disable logout for headers authenticated users (SSO)
 # studio.authentication.headers.logout.enabled: false
-# If logout is enabled for headers authenticated users (SSO), set the URL
-# studio.authentication.headers.logout.url: /mellon/logout
-# If logout is enabled for headers authenticated users (SSO), specify the HTTP method
-# studio.authentication.headers.logout.method: GET
```

Comment:
why take out the auth method?

Reply:
The SSO logout URL is now a redirect after normal logout, so it's always gonna be a GET. Specifying the method doesn't make sense anymore